verilog/tools/ls/vscode/README.md

```diff
@@ -1,18 +1,21 @@
-# VS Code Extension
+# Verible Language Server Extension
 
-## Installation
-Be sure your system as a recent version of both [nodejs](https://nodejs.org/)
-and [npm](https://www.npmjs.com/).
-From the directory of this Readme, run `npm install`.
+## Features
+The language server provides a couple of features from the Verible [SystemVerilog productivity suite](https://github.com/chipsalliance/verible) right in the editor.
 
-After [installing the verible tools](../../../README.md#installation),
-run `npm run vsix` to generate the extension file `verible.vsix`.
+ * **Linting**: checks your code against a number of
```

Comment:
s|checks|Checks| just for consistency with other bullets

Reply:
Done.